api/xds_protocol.rst

```diff
@@ -695,12 +695,12 @@ An example minimal ``bootstrap.yaml`` fragment for ADS configuration is:
             address: <ADS management server IP address>
             port_value: <ADS management server port>
         lb_policy: ROUND_ROBIN
-        http2_protocol_options: {}
-        upstream_connection_options:
-          # configure a TCP keep-alive to detect and reconnect to the admin
-          # server in the event of a TCP socket disconnection
-          tcp_keepalive:
-            ...
```

Comment:
Do we want to also demonstrate TCP keep alive and describe both options? I think TCP keep alive would work fine in many cases and is lower overhead.

Reply:
Yeah, that makes sense. I'll update.